src/modules/Host.rb

```diff
@@ -109,20 +109,18 @@ def EnsureHostnameResolvable
     # @return true if success
     def Read
       return true if @initialized
+      return true if SCR.Read(path(".target.size"), CFA::Hosts::PATH) <= 0
 
-      # read /etc/hosts
-      if Ops.greater_than(SCR.Read(path(".target.size"), CFA::Hosts::PATH), 0)
-        @hosts = CFA::Hosts.new
-        @hosts.load
-      end
+      @hosts = CFA::Hosts.new
+      @hosts.load
 
       # save hosts to check for changes later
       @hosts_init = CFA::Hosts.new
       @hosts_init.load
 
       Builtins.y2debug("hosts=#{@hosts.inspect}")
+
       @initialized = true
```

Comment:
pull request looks good, just it looks like comment for method is not longer valid, as it cannot return false. So I suggest to document that it always return true and that it can raise exception if file is malformed.